src/main/webapp/index.html

```diff
@@ -74,10 +74,30 @@ <h2>Barcode</h2>
           <p>
             Upload a barcode
           </p>
-          <p><a class="btn btn-secondary" href="#" role="button">Upload barcode photo »</a></p>
+          <p><a class="btn btn-secondary" href="#" role="button" id="upload-barcode" onclick="fileUploadDisplay()">Upload barcode photo »</a></p>
         </div><!-- /.col-lg-4 -->
+
+        <!-- UPLOAD DIV -->
+        <div class="bg-model">
+          <div class= "model-content">
+            <button class="close" onclick="closeDiv()"> + </button>
+             <h1> Upload barcode </h1>
+             <br>
```

Comment:
You seem to use `<br>` to add horizontal space, try to do this in css instead.

Reply:
Will do that!